### PR TITLE
Make testem config configurable

### PR DESCRIPTION
**Pull request by phyrog**
By default this PR changes nothing, when you are just running `ember test`. However, we can now choose which test launchers we want to run by ~~setting the `LAUNCHERS` environment variable to a space separated list of launchers (`export LAUNCHERS="Chrome Chromium"` would launch Chrome and/or Chromium, but no PhantomJS).~~ writing a local config file (see comment below).

~~Also, Launchers that were not found are ignored now (starting a test run with no launchers at all results in a successful test run, so don't do that).~~

This also configures travis to be able to run tests on Chromium ~~and Chrome as well (although only Chromium is installed). For this I set the env var `LAUNCHERS` on Travis to `"PhantomJS Chrome Chromium"`.~~

TL;DR:
- PhantomJS is now optional
- `ember test` works as normal, launching PhantomJS by default
- ~~`LAUNCHERS="Chromium Safari" ember test` launches Chromium and Safari in parallel, but not PhantomJS~~ This is configurable by a local config file.
- Travis now tests with PhantomJS and Chromium

~~Also: take a look at the `dotenv` zsh plugin if you want to specify env variables that are automatically set when entering a directory.~~

### CODE REVIEW

File: testem.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
-/*jshint node:true*/
-module.exports = {
+/* eslint-env node */
+
+let config = {
   "framework": "mocha",
   "test_page": "tests/index.html?hidepassed",
   "disable_watching": true,
+  "ignore_missing_launchers": true,
+  "parallel": 3,
```
Comment (eugenk) [inline testem.js]: Why 3?

Comment (phyrog) [inline testem.js]: Arbitrary :smile:

Comment (phyrog) [inline testem.js]: I guess we could parametrize that as well, but 3 should be good enough for now.

Comment (phyrog) [inline testem.js]: On second thought, it could be a good idea to implement something like a `testem.local.js` file where you can override values of the `testem.js` file, so we can ditch the env variables. Shouldn't be too hard.

**Comment by phyrog** [comment]
36a4238 replaces the ENV variable system by a `testem.local.js` file that can override the default config options. `testem.js` includes a short example for such a file. The gist is: the file exports a function that takes the default settings as argument (so you can e.g. push new values to an array without replacing it completely) and returns the overridden settings. This file is ignored by git.

Travis is a special case here that overrides the settings directly in the `testem.js` file.
